source/getting-started/architecture-overview.rst

```diff
@@ -182,7 +182,7 @@ The following diagram is a suggested configuration for Highly-Available database
 Load testing
 ~~~~~~~~~~~~
 
-Mattermost Enterprise Edition was `load tested <https://github.com/mattermost/mattermost-load-test>`__ with 60,000 concurrent active users with:
+Mattermost Enterprise Edition was `load tested <https://github.com/mattermost/mattermost-load-test>`__ with 60,000 concurrent activated users with:
```

Comment:
Can we confirm loadtesting was based on activated users and not of actual active users?

Reply:
@johndavidlugtu - Not sure we can with certainty. The load testing details documented on this page likely pre-dates me. How would you like to proceed with the "active/activated" terminology on this specific page?

Reply:
@nab-77 can you provide some guidance on the language here?